typescript/package.json

```diff
@@ -23,8 +23,8 @@
         "storybook": "storybook dev -p 6006",
         "build-storybook": "storybook build",
         "storybook:test": "test-storybook",
-        "docker:build": "docker build --tag wsc_story_test - < Dockerfile.test",
-        "docker:storybook:test": "docker run -it --rm --mount type=bind,source=.,target=/test --network host --ipc=host wsc_story_test",
+        "docker:build": "docker build --tag wsc_story_test --file Dockerfile.test .",
+        "docker:storybook:test": "docker run -it --rm --mount type=bind,source=./packages,target=/test/packages --network host --ipc=host wsc_story_test",
```

Comment:
Bind fails on windows with bash shell:

docker: Error response from daemon: invalid mount config for type "bind": invalid mount path: 'packages' mount path must be absolute.